### DISCUSSION TITLE
multiple folder support on navidrome

### DISCUSSION THREAD

**Discussion by shehuawwal** [navidrome/navidrome#2321] [General]
Hello team, 

I really appreciate the great work being done, I have been trying to add multiple folder to show in the navidrome, I am not sure if I do it the right way or navidrome hasn't started supporting yet.

Example:

music/2pac
music/drake
music/rihana

It combines all music, is there a way to show each of those folders, I am using docker.

**Comment by JOJ0** [comment]
Navidrome presents the imported music based on your files metadata tags only. There is no support to categorize on the basis of the originating folder: https://www.navidrome.org/docs/faq/#-can-you-add-a-browsing-by-folder-optionmode-to-navidrome

What is it you are trying to achieve? Maybe there is another way to suite your needs? You won't get folder support due to the reasons described in the link. HTH :-))
